src/main/webapp/app/course/competencies/competency-accordion/competency-accordion.component.scss

```diff
@@ -0,0 +1,94 @@
+.competency-accordion-header {
+    background-color: var(--overview-light-background-color);
+    color: var(--bs-body-color);
+    cursor: pointer;
+    padding: 5px;
+    width: 100%;
+    border: none;
+    text-align: left;
+    outline: none;
+    font-size: 15px;
+    border-radius: 3px 3px 0 0;
+    transition: 0.2s;
+}
+
+.competency-accordion-header.closed {
+    border-radius: 3px;
+}
+
+.competency-accordion-header:not([disabled]):hover {
+    background-color: rgba(100, 100, 111, 0.2);
+}
+
+.competency-accordion-ring {
+    width: 95px;
+}
+
+.competency-accordion-body {
+    padding: 0 18px;
+    background-color: var(--overview-light-background-color);
+    color: var(--bs-body-color);
+    overflow: hidden;
+    width: 100%;
+    transition:
+        max-height 0.2s ease-out,
+        padding 0.2s ease-out;
+    border-radius: 0 0 3px 3px;
+}
+
+/* Open state */
+.competency-accordion-body-open {
+    padding: 10px 18px;
+}
+
+/* Closed state */
+.competency-accordion-body-closed {
+    max-height: 0;
+    padding: 0 18px;
+}
+
+@keyframes fadeIn {
+    from {
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
+.competency-accordion-header,
```

Comment:
Codacy: `Expected selector ".competency-accordion-header" to come before selector ".competency-accordion-header:not([disabled]):hover"`